framework/source/class/qx/io/remote/Rpc.js

```diff
@@ -568,7 +568,18 @@ qx.Class.define("qx.io.remote.Rpc",
             break;
 
           case 1: // async with handler function
-            handler(result, ex, id);
+            try
+            {
+              handler(result, ex, id);
+            }
+            catch(e)
+            {
+              console.error(
```

Comment:
Yes, but 'this' isn't what we'd think it is. I could use .bind(this) on the internal function, but instead, I just assigned the context to a separate variable. Thanks.